### PR TITLE
explorer: add Pyth instruction support

### PR DESCRIPTION
**Pull request by terorie**
#### Problem

The Solana explorer shows https://github.com/pyth-network instructions as "Raw".

Pyth Network makes up a significant volume of Solana transactions.

#### Summary of Changes

Adds support for Pyth instructions.

- [x] [InitMapping](http://localhost:3000/tx/4HbNTEx3anzHw3J8x6Q5qriEoUNKHUwmr5GyUpASWVSuygFtHB8j8cYq1FfxdiYPaK4c21X3YTS3KfhQXr9YQdd7?cluster=devnet)
- [x] AddMapping
- [x] [AddProduct](http://localhost:3000/tx/ZEpDmC7DLRaPQVtJACnn8shudtHnr6EP9kPbU6rW1o2oqZd3htjaLFhcP3osoSeMyCo4EFY4Y2Y5mjoGu5uy3pa?cluster=devnet)
- [x] [UpdateProduct](http://localhost:3000/tx/31pfe7hURUY9oPieApRSUkX1LFyo5u88i26FiTPcB3mMeSy8ozwyuDmek69LunjwxggbThpRgejmgq1to1NLBGmU?cluster=devnet)
- [x] [AddPrice](http://localhost:3000/tx/31R2VBc2UBuMr7ZPfzT5W624mJJj16gheYy2AoQMcqcBx9oCvaLW4sk1jZJEcv7zzLPynj6A9hk7WTkianiyfGYW?cluster=devnet)
- [x] AddPublisher
- [x] DeletePublisher
- [x] [UpdatePrice](http://localhost:3000/tx/5itgSDnf6S9LVmdkGmTybB27FrqFthwHy5wxGY9Yo7cNgYhLTAFSoFTduafPzdeLvGEauLHqotKNa7HndUB4zpFa)
- [x] AggregatePrice
- [x] InitPrice
- [ ] InitTest
- [ ] UpdateTest
- [x] SetMinPublishers

Links direct to localhost:3000

Fixes #

### CODE REVIEW

**Comment by terorie** [comment]
Test transaction hash (Update Price), mainnet: 5itgSDnf6S9LVmdkGmTybB27FrqFthwHy5wxGY9Yo7cNgYhLTAFSoFTduafPzdeLvGEauLHqotKNa7HndUB4zpFa

File: explorer/src/components/instruction/pyth/UpdateProductDetailsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { SignatureResult, TransactionInstruction } from "@solana/web3.js";
+import { InstructionCard } from "../InstructionCard";
+import { UpdateProductParams } from "./program";
+import { Address } from "../../common/Address";
+import React from "react";
+import { Copyable } from "../../common/Copyable";
```
Comment (terorie) [inline explorer/src/components/instruction/pyth/UpdateProductDetailsCard.tsx:6]: Need to fix all these relative imports

Comment (terorie) [reply explorer/src/components/instruction/pyth/UpdateProductDetailsCard.tsx]: done

**Review by jstarry — APPROVED** [review]
Awesome, thanks!

**Pull request merged**